docusaurus/docs/create-a-plugin/develop-a-plugin/plugin-spellcheck.md

```diff
@@ -31,18 +31,18 @@ If you ended up here following the link from the failing CI then most probably y
 1. Install cspell package to your plugin's repo:
 
 ```bash
-yarn add -D cspell@6.13.3
+npm install --save-dev cspell@6.13.3
 ```
 
 3. Add `spellcheck` command to the `scripts` section in `package.json` of your plugin:
 
-```
+```json title="package.json"
 "spellcheck": "cspell -c cspell.config.json \"**/*.{ts,tsx,js,go,md,mdx,yml,yaml,json,scss,css}\""
 ```
 
 3. Create a `cspell.config.json` file in the repo root folder and add a basic config there:
 
-```json
+```json title="cspell.config.json"
```

Comment:
could sort the numbering in this list whilst here :)

Reply:
This should now be fixed. Thanks for spotting it. 👍